app/models/journeys/additional_payments_for_teaching.rb

```diff
@@ -25,7 +25,8 @@ module AdditionalPaymentsForTeaching
       "eligibility-confirmed" => EligibilityConfirmedForm,
       "correct-school" => CorrectSchoolForm,
       "reset-claim" => ResetClaimForm,
-      "postcode-search" => PostcodeSearchForm
+      "postcode-search" => PostcodeSearchForm,
+      "select-home-address" => SelectHomeAddressForm
```

Comment:
We also have `SHARED_FORMS` in the `Journeys::Base` model for forms applicable to both journeys